src/message-list/renderMessages.js

```diff
@@ -59,16 +59,11 @@ export default ({ auth, subscriptions, messages, narrow, mute, doNarrow }) => {
     list.push(
       <MessageContainer
         type="message"
-        id={item.id}
```

Comment:
We also need to change `MessageList` to look at `props.message.id` instead of `props.id` when generating its list of 'anchor' messages (otherwise scrolling gets messed up)